docs/SCD2-sparkcompute.md

```diff
@@ -18,7 +18,9 @@ int, long, float, double, bytes or string type.
 on the start date.
 
 **End Date Offset:** The offset to compute the end date. The end date will be computed as the next start date minus this offset.
-The unit is microseconds.
+The format is expected to be a number followed by an 'us', 'ms', 's', 'm', 'h', or 'd' specifying the time unit,
+with 'us' for microseconds, 'ms' for milliseconds, 's' for seconds, 'm' for minutes, 'h' for hours, and 'd' for days.
+By default, the offset is 1us.
```

Comment:
instead of mentioning the default here, it's better to just set a default in the widget spec